examples/jquery/js/app.js

```diff
@@ -173,15 +173,14 @@ jQuery(function ($) {
 			var $el = $(el);
 			var val = $el.val().trim();
 
-			if (!val) {
-				this.destroy(e);
-				return;
-			}
-
 			if ($el.data('abort')) {
 				$el.data('abort', false);
```

Comment:
add back ` return;` as it avoids an extra call to` this.render()`; which is not needed